lib/pure/fenv.nim

```diff
@@ -101,3 +101,79 @@ proc feupdateenv*(envp: ptr Tfenv): cint {.importc, header: "<fenv.h>".}
   ## Save current exceptions in temporary storage, install environment
   ## represented by object pointed to by `envp` and raise exceptions
   ## according to saved exceptions.
+
+var FP_RADIX* {. importc: "FLT_RADIX" header: "<float.h>" .} : int
```

Comment:
@Araq, I am not sure I understand. Do you mean something like this?

`````nim
var FP_RADIX_INTERNAL {. importc: "FLT_RADIX" header: "<float.h>" .} : int
                                                                                                                                                                                                                   
template FP_RADIX : int = FLT_RADIX_INTERNAL
  ## The (integer) value of the radix used to represent any floating
  ## point type on the architecture used to build the program.
`````

In this case there is no need to discriminate between `float32` and `float64`, what is the advantage of using a template?